code/data_preprocessing/genotype/VCF_QC.ipynb

```diff
@@ -139,10 +143,10 @@
     "\n",
     "```\n",
     "sos run VCF_QC.ipynb qc    \\\n",
-    "--genoFile data/MWE/MWE_genotype.vcf     \\\n",
-    "--dbsnp-variants data/reference_data/00-All.add_chr.variants.gz  \\\n",
-    "--reference-genome data/reference_data/GRCh38_full_analysis_set_plus_decoy_hla.noALT_noHLA_noDecoy_ERCC.fasta   \\\n",
-    "--cwd MWE/output/genotype_1 --container ./bioinfo.sif -J 1 -c csg.yml -q csg  &\n",
+    "    --genoFile data/MWE/MWE_genotype.vcf     \\\n",
+    "    --dbsnp-variants data/reference_data/00-All.add_chr.variants.gz  \\\n",
+    "    --reference-genome data/reference_data/GRCh38_full_analysis_set_plus_decoy_hla.noALT_noHLA_noDecoy_ERCC.fasta   \\\n",
+    "    --cwd MWE/output/genotype_1 --container ./bioinfo.sif -J 1 -c csg.yml -q csg  &\n",
```

Comment:
As discussed please dont include `-J 1 -c csg.yml -q csg  &` in MWE -- that's confusing